File: ide/bronzeData.py

from pyspark.sql import SparkSession
from pyspark.sql.functions import *
import argparse

parser = argparse.ArgumentParser()
parser.add_argument("--savePath", help="some useful description.")
parser.add_argument("--animal", help="some useful description.")
parser.add_argument("--food", help="some useful description.")
args = parser.parse_args()

# spark = SparkSession.builder.getOrCreate()

savePath = args.savePath
animal = args.animal
food = args.food
rawPath = savePath + "/raw"
bronzePath = savePath + "/bronze"
enrichedPath = savePath + "/enriched"
silverPath = savePath + "/silver"
goldPath = savePath + "/gold"

df = spark.range(10000).withColumn("colB", col("id") % 50).withColumn("colC", col("id") + 551)


# COMMAND ----------

df.write.format("delta").save(bronzePath)

File: bronzeData.scala
// Databricks notebook source
// MAGIC %run ./utils

// COMMAND ----------

import org.apache.spark.sql.functions._

val df = spark.range(10000).withColumn("colB", 'id % 50).withColumn("colC", 'id + 551)

// COMMAND ----------

display(df)

// COMMAND ----------

df.write.format("delta").save(bronzePath)
